packages/cli/src/commands/web/route/create.ts

```diff
@@ -0,0 +1,63 @@
+import { GluegunCommand } from "@webstone/gluegun";
+
+const command: GluegunCommand = {
+  alias: ["c"],
+  description: "Create a new web page",
+  run: async (toolbox) => {
+    const { parameters, print, prompt, strings, template } = toolbox;
+
+    let name = parameters.first;
+    if (!name) {
+      const result = await prompt.ask({
+        type: "input",
+        name: "name",
+        message: `What's the page name?`,
+      });
+      if (result && result.name) name = result.name;
+    }
+
+    if (!name) {
+      print.error("Please provide a page name.");
+      return;
+    }
+
+    const typesPrompt = (await prompt.ask({
+      type: "multiselect",
+      name: "types",
+      message: "What types of page do you want to create?",
+      choices: [
+        "+page.svelte",
```

Comment:
Above the first element of this `choices` array is a good place to link to Svelte's language tools repo and add a thank you note.